clusterdock/topologies/cdh/jdk/Dockerfile

```diff
@@ -12,6 +13,14 @@ RUN wget --no-check-certificate \
     tar xf /tmp/jdk.tar.gz -C /usr/java && \
     rm /tmp/jdk.tar.gz
 
+RUN apk --no-cache --update add unzip
+
+RUN wget --no-check-certificate \
+    --no-cookies --header "Cookie: oraclelicense=accept-securebackup-cookie" -O /tmp/unlimited_jce_policy.zip \
```

Comment:
Nit: when you have line continuations for the same command and `&&`-separated commands, indent the line continued same commands by 4 more spaces so that it's easier to tell where one command ends. e.g.
```
RUN wget --no-check-certificate \
        --no-cookies ... && \
    unzip ... && \
    rm ...
```